tcp/src/lib.rs

```diff
@@ -28,486 +28,41 @@
 //! use std::str::FromStr;
 //!
 //! fn main() {
-//! 	let mut io = IoHandler::new();
+//! 	let mut io = MetaIoHandler::<()>::new();
 //! 	io.add_method("say_hello", |_params| {
 //! 		Ok(Value::String("hello".to_string()))
 //! 	});
-//! 	let server = Server::new(&SocketAddr::from_str("0.0.0.0:9993").unwrap(), io).unwrap();
-//!     ::std::thread::spawn(move || server.run());
+//! 	let server = Server::new(SocketAddr::from_str("0.0.0.0:9993").unwrap(), Arc::new(io));
+//!     ::std::thread::spawn(move || server.run().expect("Server must run with no issues"));
 //! }
 //! ```
 
-extern crate jsonrpc_core;
+extern crate jsonrpc_core as jsonrpc;
 #[macro_use] extern crate log;
 #[macro_use] extern crate lazy_static;
 extern crate env_logger;
```

Comment:
Should be used only in tests (maybe even do `extern crate env_logger` in `mod logger`)